pkg/constant/constants.go

```diff
@@ -0,0 +1,7 @@
+package constant
+
+const (
+	AnnoHypershiftDeployment = "hypershift.open-cluster-management.io/hypershiftdeployemnt"
```

Comment:
The APIGroup for `HypershiftDeployment` is `cluster.open-cluster-management.io`.  So we should probably use that here.

Reply:
ok, I will sync it here https://github.com/stolostron/backlog/issues/20056 , too.